serverless/messaging/concepts.mdx

```diff
@@ -139,13 +150,21 @@ Stream persistence is one of the factors affecting the billing of Scaleway Messa
 
 In [publish/subscribe](#publishsubscribe) systems such as [SNS](#sns), a subscriber is the entity (e.g. SQS queue, function, URL) that messages from topics are pushed to. Subscribers can filter messages based on their topic or content.
```

Comment:
Not sure about this one, but shouldn't there be a slash between "publish" and "subscribe" in the anchor?

Reply:
No, I checked